lib/koans/15_protocols.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Protocols do
+  use Koans
+
+  defprotocol School do
+    def enrol(name)
```

Comment:
[Double 'l'  is the American spelling](http://grammarist.com/spelling/enrol-enroll/) and else where one 'l' is usedI am happy to change to use the American spelling if preferred.